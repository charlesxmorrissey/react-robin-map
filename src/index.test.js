import ReactDOM from 'react-dom'

jest.mock('react-dom', () => ({ render: jest.fn() }))

describe('Application', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div')
    div.id = 'root'
    document.body.appendChild(div)
    require('./index.js')
    expect(ReactDOM.render).toMatchSnapshot()
  })
})
