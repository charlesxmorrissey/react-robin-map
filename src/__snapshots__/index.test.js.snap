// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application should render without crashing 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      <Provider
        store={
          Object {
            "dispatch": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "runSaga": [Function],
            "subscribe": [Function],
            Symbol(observable): [Function],
          }
        }
      >
        <BrowserRouter>
          <App />
        </BrowserRouter>
      </Provider>,
      <div
        id="root"
      />,
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
